.travis.yml

```diff
@@ -1,36 +1,55 @@
 language: python
 
-sudo: false
-
-python:
-  - 3.5
-
-env:
-  - export FAIL_ON_EXTERNAL_DEPRECATION='False'
-  - export FAIL_ON_EXTERNAL_DEPRECATION='True'
-
 matrix:
+  include:
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
```

Comment:
@vidartf, do you mean to remove osx with failure_by_deprecation=true, and leave only 3 instances in the matrix? so that deprecations would be checked only on linux?